### PR TITLE
Add yarn install

### PR DESCRIPTION
**Pull request by AkiaCode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@
 - [License](#license) 
 
 ## Install
-`npm install loggaby`
+`npm install loggaby` or `yarn add loggaby`
```
Comment (sammy-ette) [inline README.md]: it should say "Or with Yarn:" instead of just "or `<yarn install>`"

**Review by sammy-ette — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AkiaCode) [reply README.md]: Okay

Comment (AkiaCode) [reply README.md]: ex) `Or with Yarn: yarn add loggaby`

@TorchedSammy  right?

Comment (sammy-ette) [reply README.md]: yes, but please newline after `:`

Comment (AkiaCode) [reply README.md]: `Or with Yarn: `
`yarn add loggaby`

Comment (sammy-ette) [reply README.md]: 👌 yup

**Review by sammy-ette — APPROVED** [review]
you can also newline with just 2 spaces at the end of a line

**Comment by AkiaCode** [comment]
> you can also newline with just 2 spaces at the end of a line

Okay

**Pull request merged**